src/assertions/contains.rs

```diff
@@ -0,0 +1,23 @@
+use colored::*;
+use std::fmt::Debug;
+
+pub fn assert_contains<V: PartialEq + Debug, I: IntoIterator<Item = V> + Debug + Clone>(
+    left: I,
+    right: V,
+) -> Option<String> {
+    if left.clone().into_iter().any(|item| item == right) {
```

Comment:
definitely can't do `.clone()` here, it can be too expensive!
you can just use a for loop without having to clone anything